source/lifecycle/mattermost-v9-changelog.md

```diff
@@ -0,0 +1,991 @@
+# Mattermost v9 changelog
+
+```{Important}
+**Support for Mattermost Server v8.1 Extended Support Release has come to the end of its life cycle on May 15, 2024. Upgrading to Mattermost Server v9.5 or later is required.**
+- See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) documentation for details on upgrading to a newer release.
+- See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
+```
+
+(release-v9-8-feature-release)=
```

Comment:
@amyblais - Can I get your help to add the v9.9 draft changelog content here, please?